ktor-shared/ktor-htmx/common/src/io/ktor/htmx/HxAttributeKeys.kt

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+@file:Suppress("ConstPropertyName")
+
+package io.ktor.htmx
+
+public object HxAttributeKeys {
+    /**
+     * issues a GET to the specified URL
```

Comment:
Here and in some other files KDocs are starting with a lowercase letter. Could you fix it?
I also think that we could add a link to the official docs to the KDoc of this class: https://htmx.org/reference/#attributes-additional